docker/prod/uwsgi/uwsgi.service

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+if [ "${CONTAINER_NAME}" = "listenbrainz-web" ]
+then
+    exec uwsgi /etc/uwsgi/uwsgi.ini
+fi
+if [ "${CONTAINER_NAME}" = "listenbrainz-influx-writer" ]
+then
+    cd /code/listenbrainz
+    exec python influx-writer/influx-writer.py
+fi
+if [ "${CONTAINER_NAME}" = "listenbrainz-bigquery-writer" ]
+then
+    cd /code/listenbrainz
+    exec python bigquery-writer/bigquery-writer.py
+fi
+if [ "${CONTAINER_NAME}" = "listenbrainz-alpha-importer" ]
+then
+    cd /code/listenbrainz
+    exec python alpha_importer.py
+fi
+
+echo "init script has no clue which service to start. Set env var CONTAINER_NAME!"
```

Comment:
exec without redirections, replaces this script in memory -- there are no further statements executed in this script after exec.

And this script is the entry point for the container -- this would be a massive chicken-and-the-egg problem that I really don't wish to tackle. Especially with us possibly moving to swarm, all this might change.